### ISSUE TITLE
Can we use the pre-training model directly on the data set with added categories?

### ISSUE THREAD

**Issue by divided7** [ultralytics/yolov5#10969]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Can we use the pre-training model directly on the data set with added categories?
Such as we have a dataset: number of label classes is 80(pretrained by coco) + 20(custom data) = 100 cls,
can we direct use : python train.py --weights yolov5s.pt ?

**Comment by JustasBart** [comment]
@divided-by-7 Yes, absolutely! But be aware that in order for you to train this new model of 100 classes you need the COCO dataset as well as your additional dataset to be training all at once. You can't not show it the COCO dataset and then only train it on your own data unless if you are willing to have two models running concurrently.

I hope this help, good luck! :rocket:

**Comment by divided7** [comment]
> @divided-by-7 Yes, absolutely! But be aware that in order for you to train this new model of 100 classes you need the COCO dataset as well as your additional dataset to be training all at once. You can't not show it the COCO dataset and then only train it on your own data unless if you are willing to have two models running concurrently.
> 
> I hope this help, good luck! 🚀

Thank you for your help! 
I wonder can we add Conv Core on some layer? 
Such as we got tensor : shape = [ [batch size,255(=85 * 3),20,20], [batch size,255(=85 * 3),40,40], [batch size,255(=85 * 3),80,80] ] by Conv2d(xx,255,...) in backbone outputs, the Conv2d Core(weights) shape is [255, xxx , 3,3], we freeze this Core,and add more core (in Conv2d weights.shape[0]),such as we need 100cls : Conv2d.weights = torch.cat(Conv2d.weights, tensor([20 * 3,xxx,3,3]), dim = 0 ), and we freeze the first 255 layers, only train the last 20 * 3 layers?
I want to achieve that train and get 100cls model only use additional 20cls custom dataset.
